typescript/packages/jumble/src/views/Shell.tsx

```diff
@@ -24,6 +26,14 @@ export default function Shell() {
   const onLaunchCommand = useCallback(() => {
     window.dispatchEvent(new CustomEvent("open-command-center"));
   }, []);
+  
+  const { user } = useAuthentication();
+
+  if (!user) {
+    return (
+      <AuthenticationView /> 
+    )
+  }
```

Comment:
Would be better ultimately to use an effect to perform a redirect here but this is fine for now.